src/crypto/SecretKey.cpp

```diff
@@ -149,6 +150,20 @@ SecretKey::random()
     return sk;
 }
 
+SecretKey
```

Comment:
should we ifdef this stuff with `BUILD_TESTS` or move to a test only file?

Reply:
Oh, perhaps. Will look at that.

Reply:
Fixed.